css-three-column-layout/styles.css

```diff
@@ -0,0 +1,61 @@
+body {
+  font-family: Lato, sans-serif;
+  background-color: #1996da;
+  color: #fff;
+  text-align: center;
+}
+
+.container {
+  max-width: 1100px;
+  padding-top: 50px;
+}
+
+.row {
+  display: flex;
+}
+
+.centered {
+  justify-content: center;
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+  padding-left: 1.5rem;
+  padding-right: 1.5rem;
+  margin-top: 10px;
```

Comment:
Just `flex-basis` on `column-` classes. Move anything else to another class that you can add to the element in addition to the `column-` class

Reply:
Updated to reflect changes.
<img width="301" alt="image" src="https://user-images.githubusercontent.com/112663686/195963083-a5acc8ec-55d2-4fbf-ac9b-b0fac602a445.png">